### ISSUE TITLE
Model training varies greatly each time its run.

### ISSUE THREAD

**Issue by kegro** [charlotte-pel/igarss2019-dl4sits#1]
Thanks for sharing, and nice paper! 
I've tried training the deep learning models a few times using the data and default settings you provide.  Each training run ends after between 2 and 9 epochs, and not the 20 n_epochs defined. The accuracy of each model also varies greatly each time.
Can you tell me 1. why training stops like this? and 2. Is there a way to train so that a consistently high validation accuracy is reached?
I have some similar crop time series I would love to try this on.

**Comment by kegro** [comment]
I found a solution. I increased the `patience` for the early stopping:
https://github.com/charlotte-pel/igarss2019-dl4sits/blob/b0311a55a0db5c9550acd49bcd0b91e56ad92df8/dl_func.py#L247
